ngraph/frontend/onnx_import/src/op/constant.cpp

```diff
@@ -42,9 +42,14 @@ namespace ngraph
                         }
                         catch (const ngraph::ngraph_error& exc)
                         {
-                            NGRAPH_WARN << "Could not create an nGraph Constant for an ONNX "
-                                           "Constant node. Detailed error:\n"
-                                        << exc.what();
+                            NGRAPH_WARN
```

Comment:
I suggest to additionally switch to `NGRAPH_INFO` so that it's less scary for the users.

Reply:
No problem for me, but I can't see any usage of `NGRAPH_INFO` in onnx importer so we usually communicate such things by warning.  @postrational what do you think?

Reply:
If a model with no tensor data is a behaviour documented in the ONNX spec, this should be log level `INFO`, could even be `DEBUG`. Otherwise, let's stick with `WARN`.

Reply:
As far as I can tell, this is not documented anywhere, so we should keep a warning here.
https://github.com/onnx/onnx/blob/490895703c82f68e723a8b308369cac4ed0abc0c/onnx/onnx.proto#L547-L548